addons/udes_stock/models/stock_warehouse.py

```diff
@@ -56,6 +56,21 @@ def _domain_u_damaged_location_id(self):
         help="Allowed tracking types on product level, values are comma separated and can use any combination of 3 "
              "possible options none, lot,serial."
     )
+    u_force_upper_case_config = fields.Text(
```

Comment:
suggestion: use a many2many table of `model_id` and `field_ids` instead of text, so we do not need to handle json parse errors